src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -27,6 +27,7 @@ internal class ConsoleGui : IDisposable
         private int _listViewOffset;
         private Label _filterErrorLabel;
         private Dictionary<int, int> _indexMap = new Dictionary<int, int>();
+        private List<int> _selectedIndexes = new List<int>();
```

Comment:
If the object thing doesn't work out for whatever reason... at least make this a `HashSet` so that the `.Contains()` is O(1) instead of O(N)

Reply:
Will change